ray-operator/test/e2eautoscaler/create_detached_actor_submit_task.py

```diff
@@ -0,0 +1,36 @@
+"""This script create an actor that will continuously submits tasks to simulate user workload"""
+
+import ray
+import sys
+import argparse
+import time
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--num-cpus", type=float, default=1)
+parser.add_argument("--num-gpus", type=float, default=0)
```

Comment:
My philosophy is only adding something when it is actually used. For example, `--num-gpus` is not used here, so I will remove it.